ush/cam/evs_href_snowfall.sh

```diff
@@ -16,14 +16,14 @@ mkdir -p $COMOUTsmall/HREF_SNOW
 #NOHRSC data missing alert
 if [ ! -s $COMSNOW/${VDATE}/wgrbbul/nohrsc_snowfall/sfav2_CONUS_24h_${VDATE}12_grid184.grb2 ] ; then 
   if [ $SENDMAIL = YES ] ; then
-   echo "Warning:  No NOHRSC data available for ${VDATE}"
+   echo "WARNING:  No NOHRSC data available for ${VDATE}"
    export subject="NOHRSC Data Missing for EVS ${COMPONENT}"
-   echo "Warning:  No NOHRSC data available for ${VDATE}" > mailmsg
+   echo "WARNING:  No NOHRSC data available for ${VDATE}" > mailmsg
    echo Missing file is  $COMSNOW/${VDATE}/wgrbbul/nohrsc_snowfall/sfav2_CONUS_24h_${VDATE}12_grid184.grb2  >> mailmsg
    echo "Job ID: $jobid" >> mailmsg
    cat mailmsg | mail -s "$subject" $MAILTO
   fi
-   echo "Warning:  No NOHRSC data available for ${VDATE}! Terminate snowfall verification"
+   echo "WARNING:  No NOHRSC data available for ${VDATE}! Terminate snowfall verification"
    exit
 fi
```

Comment:
We need a the WARNING statement in line 26 to have the full path of the missing file.